samples/SimpleUnitTest/SimpleTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace SimpleUnitTest
+{
+    [TestFixture]
+    public class SimpleTests
+    {
+        [Test]
+        public void Passing_test()
+        {
+            Assert.Pass();
+        }
+        //---------------------------------------------------------------------
+        [Test]
+        public void Failing_test()
+        {
+            Assert.Fail("Failing for demo purposes");
```

Comment:
Comment that in the test-scripts `set +e` is needed, so that the step won't fail in CI.